src/wallet3/cli-wallet/oxen_wallet_cli/walletcli.py

```diff
@@ -0,0 +1,110 @@
+import logging
+import os
+
+import click
+from click_repl import repl
+
+from oxen_wallet_cli import context
+
+import pywallet3
+
+def _get_config_dir(options):
+    """Return the default config dir for network"""
+    return os.path.expanduser(os.path.join('~', '.oxen-wallet', options['network']))
+
+@click.group(invoke_without_command=True)
+@click.option('--log-level', type=click.Choice(['error', 'warn', 'info', 'debug']))
+@click.option('--network', default='testnet', help='Network: mainnet|testnet|devnet.')
+@click.option('--config-dir', '-C', default=None, help='Override config directory.')
+@click.option('--oxend-url', default="ipc:///home/sean/.oxen/testnet/oxend.sock", type=str, help='Use the given daemon')
+@click.option('--datadir', help='A directory which gdk will use to store encrypted data relating to sessions')
```

Comment:
gdk?

Reply:
yeah not us. from the forked wallet code